src/handler.py

```diff
@@ -135,7 +154,266 @@ def restore_db_cluster(event, context):
+    rds = RDS(db_host, db_user, db_name, db_password)
+    sql = "select count(1) from capture.json_data"
+    result = rds.execute_sql(sql)
+    logger.info(f"RESULT: {result}")
+
+    content = {"StartTime": datetime.datetime.now(), "StartCount": result}
+    logger.info(f"Writing this to S3 {json.dumps(content)}")
+    s3.Object('iow-retriever-capture-load', 'TEST_RESULTS').put(Body=json.dumps(content))
+
+
+def falsify_secrets(event, context):
+    """
+    Todo replace lambda environment variables for password and db address
+    while leaving others untouched.
+
+    1. call lambda_client.get_function_configuration() to get original env variables
+    2. call secrets_client.get_secret_value() to get the secrets we want to change
+    3. replace the env variable for "AQTS_SCHEMA_OWNER_PASSWORD" or "TRANSFORM_SCHEMA_OWNER_PASSWORD"
+       with the new password
+    4. replace the env variable for "AQTS_DATABASE_ADDRESS" or "TRANSFORM_DATABASE_ADDRESS"
+       with the new database address
+    5. update the environment variables
+    :param event:
+    :param context:
+    :return:
+    """
+
+    original = secrets_client.get_secret_value(
+         SecretId=NWCAPTURE_LOAD
+    )
+    secret_string = json.loads(original['SecretString'])
+    db_password = str(secret_string['SCHEMA_OWNER_PASSWORD'])
+    db_address = str(secret_string['DATABASE_ADDRESS'])
+    logger.info(f"db_address {db_address} db_password {db_password}")
+
+    for lambda_function in LAMBDA_FUNCTIONS:
+        # 1.
+        response = lambda_client.get_function_configuration(
+            FunctionName=lambda_function
+        )
+        my_env_variables = response['Environment']['Variables']
+        logger.info(f"BEFORE function {lambda_function} my_env_variables= {my_env_variables}")
+        if my_env_variables.get("AQTS_SCHEMA_OWNER_PASSWORD") is not None:
+            my_env_variables["AQTS_SCHEMA_OWNER_PASSWORD"] = db_password
+        elif my_env_variables.get("TRANSFORM_SCHEMA_OWNER_PASSWORD") is not None:
+            my_env_variables["TRANSFORM_SCHEMA_OWNER_PASSWORD"] = db_password
+        if my_env_variables.get("AQTS_DATABASE_ADDRESS") is not None:
+            my_env_variables["AQTS_DATABASE_ADDRESS"] = db_address
+        elif my_env_variables.get("TRANSFORM_DATABASE_ADDRESS") is not None:
+            my_env_variables["TRANSFORM_DATABASE_ADDRESS"] = db_address
+        if my_env_variables.get("DB_PASSWORD") is not None:
+            my_env_variables["DB_PASSWORD"] = db_password
+        if my_env_variables.get("DB_HOST") is not None:
+            my_env_variables["DB_HOST"] = db_address
+        logger.info(f"AFTER function {lambda_function} my_env_variables= {my_env_variables}")
+
+        # lambda_client.update_function_configuration(
+        #     FunctionName=lambda_function,
+        #     Environment={
+        #         'Variables': my_env_variables
+        #     }
+        # )
+
+
+def restore_secrets(event, context):
```

Comment:
I wonder if `restore_secrets` and `falisfy_secrets` can be the same function... :thinking:

Reply:
I will make that change in the next PR.  We are going to have a meeting with Jim about what tier this should run on, and right now this is hardcoded to run on test, so there is at least one more PR coming.